src/arche/tools/schema.py

```diff
@@ -73,4 +76,15 @@ def basic_json_schema(data_source: str, items_numbers: List[int] = None):
         items_numbers: array of item numbers to create schema from
     """
     schema = create_json_schema(data_source, items_numbers)
-    print(json.dumps(schema, indent=4))
+    return BasicSchema(schema)
+
+
+@dataclass
+class BasicSchema:
+    schema: Schema
+
+    def json(self):
+        print(json.dumps(self.schema, indent=4))
+
```

Comment:
maybe a method `dict()` to make the API it similar to `json()`.

Reply:
I decided to leave that out since we can use `d` field - BasicSchema().d